src/selfplay/loop.cc

```diff
@@ -123,13 +126,46 @@ void SelfPlayLoop::CmdSetOption(const std::string& name,
 }
 
 void SelfPlayLoop::SendTournament(const TournamentInfo& info) {
+  int winp1   = info.results[0][0] + info.results[0][1];
+  int loosep1 = info.results[2][0] + info.results[2][1];
+  int draws   = info.results[1][0] + info.results[1][1];
+  float perct = -1, elo = 99999;
```

Comment:
what's perct?
Why 99999?
Also, should be two separate lines.

Reply:
perct is win percentage. Will change.